.github/workflows/ci-test.yaml

```diff
@@ -8,24 +8,24 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: 1.15
-      - name: make local-test
+      - name: Run unit tests
         env:
           VERBOSE: true
         run: |
-          make local-test
+          source ./scripts/build.sh; unit_local
   stress-test-linux:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: make stress-test
+      - name: Run stress tests
         env:
           VERBOSE: true
         run: |
-          make stress
+          source ./scripts/build.sh; stress
   integration-test-on-kind:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - name: run tests
         run: |
-          SONOBUOY_CLI=../../build/linux/amd64/sonobuoy ./scripts/run_integration_tests.sh
+          source ./scripts/build.sh; prepare_and_run_integration
```

Comment:
Using source for now; will probably add a handler for arguments so that we can call it directly. I ran into a silly problem where I was testing this all locally and my terminal kept closing. But it was because of the source & set -e, then on failure my own terminal would exit :) Took me longer than necessary to figure that one out.

I'll go ahead and start working on the handler code, it can be in this PR or the next depending on when you review/when I'm done.

Reply:
I've gone back and forth on this. Sourcing may end up being the best way since the alternatives are:
 - a separate "cli" in bash to ensure we call into the functions properly. This ends up being nearly as long as the file itself since we have so many short functions
 - a go wrapper with the same pitfalls as the above

At least with the script as is, it is just a single, ubiquitous language and we don't have to bounce around to figure out whats going on or google (as much) esoteric language/edge cases.

Just shouldn't source into your own shell usually :) Happy to discuss and bounce ideas around if there's something better. It really is just for devs+ci so I think it'll be fine regardless of what we do.